dev/build.image/profiles/jakartaee9/features.xml

```diff
@@ -1,5 +1,9 @@
 <feature>jakartaee-9.0</feature>
 <feature>jakartaeeClient-9.0</feature>
 <feature>appSecurityClient-1.0</feature>
+<feature>monitor-1.0</feature>
```

Comment:
Would you like to add `jakartaee-9.1` and `jakartaeeClient-9.1` to this package now?

Reply:
I would like to do that in a separate PR if that is ok.  Need to work out whether to add jakartaee-9.1 or replace 9.0 with 9.1.  I was thinking this PR would unblock your PR #17582 so that it could be used to deliver that change.